thirdstage-react/src/component/About/about.jsx

```diff
@@ -0,0 +1,22 @@
+import "./aboutStyle.scss"
+import Card from "../cards/AboutCard/aboutCards"
+const array=["7.jpg","8.jpg","9.jpg","7.jpg",]
```

Comment:
add it inside About function

Reply:
i searched about the best declaration for it and that what i got 
If the array doesn't need to be recreated every time the component renders, declaring it outside the component can improve performance. Constantly recreating the array inside the component can lead to unnecessary work.
and in this case its static